src/bentoml/_internal/server/http/instruments.py

```diff
@@ -146,7 +162,7 @@ async def wrapped_send(message: "ext.ASGIMessage") -> None:
                         service_name=component_context.bento_name,
                         service_version=component_context.bento_version,
                         http_response_code=STATUS_VAR.get(),
-                    ).inc()
+                    ).inc(exemplar=exemplar)
```

Comment:
How do we control the sampling? Does the current implementation include all requests as examplars?